zebedee-cms/src/main/java/com/github/onsdigital/zebedee/ZebedeeConfiguration.java

```diff
@@ -83,20 +78,7 @@
 import static com.github.onsdigital.zebedee.Zebedee.USERS;
 import static com.github.onsdigital.zebedee.Zebedee.ZEBEDEE;
 import static com.github.onsdigital.zebedee.configuration.CMSFeatureFlags.cmsFeatureFlags;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getDatasetAPIAuthToken;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getDatasetAPIURL;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getImageAPIURL;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getKafkaContentPublishedTopic;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getKafkaURL;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getKeyringInitVector;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getKeyringSecretKey;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getServiceAuthToken;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getSlackSupportChannelID;
-import static com.github.onsdigital.zebedee.configuration.Configuration.slackChannelsToNotfiyOnStartUp;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getIdentityAPIURL;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getMaxRetryInterval;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getMaxRetryTimeout;
-import static com.github.onsdigital.zebedee.configuration.Configuration.getInitialRetryInterval;
+import static com.github.onsdigital.zebedee.configuration.Configuration.*;
```

Comment:
Please return the single import 
(if you are using intellij push and commit in terminal outside ide)

Reply:
OK. Is there. a standaed IntelliJ formatter settings file that should be used?